openverse-api/catalog/api/models/media.py

```diff
@@ -101,6 +101,9 @@ class Meta:
         Meta class for all media types indexed by Openverse. All concrete media
         classes should inherit their Meta class from this.
         """
+        unique_together = [
```

Comment:
Django docs advises to use [UniqueConstraints](https://docs.djangoproject.com/en/3.2/ref/models/options/#unique-together) instead of using unique_together which may be deprecated in the future.

Reply:
Oh, they must have introduced this recently. Thanks for the reference. I'll make this change and update the PR.

Reply:
Updated in f2381ec.